### PR TITLE
migrate/remove inlineMixin  and rmrkVersionMixin 

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Refactoring

## Context

- [x] Related #6392 migrate/remove inlineMixin  and rmrkVersionMixin 
- [x] Related #4750 reduce using nuxt-property-decorator

### CODE REVIEW

File: composables/useRmrkVersion.ts [hunk 1]
```diff
@@ -3,14 +3,14 @@ import { RemarkVersion } from '@kodadot1/minimark/shared'
...
-  const version = computed<RemarkVersion | null>(() => {
+  const version = computed<RemarkVersion>(() => {
     switch (urlPrefix.value) {
       case 'rmrk':
         return '1.0.0'
       case 'ksm':
         return '2.0.0'
       default:
-        return null
+        return '2.0.0'
```
Comment (daiagi) [inline composables/useRmrkVersion.ts:13]: why?

Comment (Jarsen136) [reply composables/useRmrkVersion.ts]: I originally wanted to keep the same implementation as on the mixin, but now I think it's not that suitable.

**Review by daiagi — APPROVED** [review]
lgtm

**Review by preschian — APPROVED** [review]

File: components/shared/LinkResolver.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <component :is="is">
```
Comment (vikiival) [inline components/shared/LinkResolver.vue:2]: hmm ok

**Pull request merged**